docs/README.md

```diff
@@ -2,63 +2,12 @@
 
 ### Getting Started
 
-For set-up information and running your first Workflows, please see our [Getting Started](getting-started.md) guide.
+For set-up information and running your first Workflows, please see our [Getting Started](quick-start.md) guide.
 
 ### Examples
 
-For detailed examples about what Argo can do, please see our [Argo Workflows: Documentation by Example](../examples/README.md) page.
+For detailed examples about what Argo can do, please see our [documentation by example](examples/README.md) page.
 
 ### Fields
 
 For a full list of all the fields available in for use in Argo, and a link to examples where each is used, please see [Argo Fields](fields.md).
-
-### Features
-Some use-case specific documentation is available:
-
-* [Contributing](CONTRIBUTING.md)
-* [Access Token](access-token.md)
```

Comment:
Where is this going to go? Is it getting removed entirely?

Reply:
mkdocs will create an index from on `mkdoc.yaml`